orc8r/cloud/go/services/configurator/storage/sql_entity_load_helpers.go

```diff
@@ -129,7 +142,12 @@ func (store *sqlConfiguratorStorage) getBuilder(networkID string, filter EntityL
 	}
 	isInNetwork := sq.Eq{entCol(entNidCol): networkID}
 
-	cols := getLoadEntitiesCols(criteria)
+	var cols []string
+	if loadTyp == countEntities {
+		cols = []string{"COUNT(1)"}
+	} else {
+		cols = getLoadEntitiesCols(criteria)
+	}
 	// If we're loading assocs: don't need to load everything
 	if loadTyp == loadChildren || loadTyp == loadParents {
 		cols = getLoadAssocCols()
```

Comment:
Make this one big switch on loadTyp -- right now it's two separate if statements